code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -108,6 +104,12 @@
 	if(user.s_active)
 		user.s_active.hide_from(user)
 
+	//We re-orient our items every time we want to show them to someone. Technically, this is wasteful, and it would be more efficient to rebuild this only when adding/removing items from us.
+	//However, most of the (thousands of) instances of items being spawned inside containers simply use the unsafe method of directly new()ing the thing into the content list.
+	//This does not use any helpers whatsoever. In fact, according to BYOND docs, it's by design that spawning new items inside us does not call Entered() or leave any trace at all. (thanks BYOND!)
```

Comment:
Yeah, BYOND doesn't call `Entered()` when a new object is created in something, but we could. Honestly I thought we already did. We call `Exited()` when an object is deleted, after all.